boards/nucleo-f429zi/include/periph_conf.h

```diff
@@ -46,10 +47,12 @@ extern "C" {
 static const dma_conf_t dma_config[] = {
     { .stream = 11 },   /* DMA2 Stream 3 - SPI1_TX */
     { .stream = 10 },   /* DMA2 Stream 2 - SPI1_RX */
+    { .stream = 8 },    /* DMA2 Stream 0 - ETH_TX  */
```

Comment:
TODO for myself: Check if this is actually still used and drop for all boards consistently if not. If I recall correctly, the previous version of the driver used this to speed up memcpy to a TX buffer. But I rewrote the driver to use the ETH DMA to directly collect tje data to transmit in a zero-copy fashion. If I recall correctly, the DMA stream can be used for other stuff, as the ETH DMA is a distinct DMA unit.